msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -54,11 +54,28 @@ Create the conversation, after you have the user or channel information.
 
 ## Create the conversation
 
-Create the conversation if it doesn't exist or you don't know the `conversationId`. Create the conversation only once and store the `conversationId` value or `conversationReference` object.
+You can create the conversation if it doesn't exist or you don't know the `conversationId`. Create the conversation only once and store the `conversationId` value or `conversationReference` object.
+
+To create a conversation, you need `userId`, `tenantId`, and `serviceUrl`.
+
+For `serviceUrl`, use the value from an incoming activity triggering the flow or one of the global service URLs. The following is an example for the `serviceUrl`:
+
+```csharp
+const serviceUrl = 'https://smba.trafficmanager.net/uk/';
+        `MicrosoftAppCredentials.trustServiceUrl(serviceUrl);
+```
+
+If the `serviceUrl` isn't available from an incoming activity triggering the proactive scenario, use the following global URL endpoints:
+
+* Production: `https://smba.trafficmanager.net/teams/`
+* GCC: `https://smba.infra.gcc.teams.microsoft.com/gcc`
+* GCCH: `https://smba.infra.gov.teams.microsoft.us/gcch`
+
+Code sample to [**create a conversation bot**](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/57.teams-conversation-bot/Bots/TeamsConversationBot.cs).
```

Comment:
Note that the code in this sample is incorrect. The "channel ID" here should be "msteams", not the Teams channel ID.
<img width="437" alt="image" src="https://user-images.githubusercontent.com/767208/195409880-da616080-441c-4d3d-8e76-d81ecb30bf97.png">

Reply:
I will try to update the sample. Will add a note here when done

Reply:
created the PR to fix the bug in the bot sample
https://github.com/microsoft/BotBuilder-Samples/pull/3833

Reply:
the pr for the fix of the sample bot was merged